draft-ietf-jsonpath-base.md

```diff
@@ -1592,6 +1663,54 @@ Change controller:
 Provisional registration? (standards tree only):
 : no
 
+## Function Extensions {#iana-fnex}
+
+This specification defines a new "Function Extensions sub-registry" in
+a new "JSONPath Parameters registry", with the policy "expert review"
+({{Section 4.5 of -ianacons}}).
+
+The expert is instructed to be frugal in the allocation of very short
+function extension names, keeping them in reverse for applications that
+are likely to enjoy wide use and can make good use of their shortness.
```

Comment:
Not sure about "can make good use of their shortness". When would a short name not be made good use of?

Reply:
Indeed, this is really less about shortness.  Say, someone registers a size() that extracts the shirt size of a JSON description of a shirt.

Reply:
Actually, why not simply the following?
```suggestion
are likely to enjoy wide use.
```
You see I'm not sure the phrase "and can make good use of their shortness" is anything more than motherhood.

Reply:
Addressed in 85e5525